src/Label/Label.jsx

```diff
@@ -0,0 +1,104 @@
+import _ from "lodash";
+import classnames from "classnames";
+import React, {PropTypes, PureComponent} from "react";
+
+import {Tooltip} from "../Tooltip";
+
+import "./Label.less";
+
+
+export default class Label extends PureComponent {
+  focus() {
+    if (this.refs.label) {
+      this.refs.label.focus();
+    }
+  }
+
+  render() {
+    const {cssClass} = Label;
+    const {
+      children,
+      className,
+      color,
+      size,
+      tooltip,
+      tooltipPlacement,
+      tooltipTextAlign,
+      ...passthroughProps,
+    } = this.props;
+
+    const label = (
+      <span
+        className={classnames(
+          cssClass.CONTAINER,
+          color && cssClass.color(color),
+          size && cssClass.size(size),
+          tooltip && cssClass.WITH_TOOLTIP,
+          className
+        )}
+        ref="label"
+        tabIndex={tooltip ? 0 : undefined}
+        {...passthroughProps}
+      >
+        {children}
+      </span>
+    );
+
+    if (!tooltip) {
+      return label;
+    }
+
+    return (
+      <Tooltip content={tooltip} placement={tooltipPlacement} textAlign={tooltipTextAlign}>
+        {label}
+      </Tooltip>
+    );
+  }
+}
+
+Label.Color = {
```

Comment:
Thinking to the future, I feel like we should make these just non-default exported consts rather than members of the actual Label class, because Typescript will almost certainly yell at us for assigning nonexistent members; and I don't think it's any more clear this way either.

But given we're not doing that yet, and I don't think clients really consume this stuff, it's no big deal